ipynb/releases/ReleaseNotes_v16.12.ipynb

```diff
@@ -0,0 +1,778 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "heading_collapsed": true
+   },
+   "source": [
+    "# Target Connectivity"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "heading_collapsed": true
+   },
+   "source": [
+    "## Configurable logging system"
```

Comment:
I had the same doubt, but at the same time I would like to keep a consistent set of main sections for all ReleaseNotes. Thus, I've put it under Target connectivity since most of the time we need logging to debug what's going on on the target... Do you suggest another main section?